Platform/TigerlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -58,6 +58,116 @@ GetBoardId (
   OUT UINT8 *BoardId
 );
 
+
+/* BAD DSO mark:
+     The 'Signature' and 'Size' fields are zero(0).
+     Other fields, like Svn, are subject to change
+  */
+STATIC LOADER_COMPRESSED_HEADER mBadDsoMark = {0};
```

Comment:
For Stage1B, we cannot use global variable.  The reason is that Stage1B could be running from Flash.
Please check if we can avoid using this global variable.

Reply:
Thanks and will be updated.